.env.prod

```diff
@@ -17,10 +17,7 @@ REACT_APP_ETHEREUM_NETWORK_ID="1"
 REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Mainnet"

```

Comment:
Why did you skip `*SUPPORT*` for PROD?

Reply:
I will update all the prod values in a separate PR. Once dev and stage is tested.